dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/ProbeExtensionArchive.java

```diff
@@ -177,8 +178,8 @@ public boolean isExtClassesOnly() {
 
     /** {@inheritDoc} */
     @Override
-    public Set<String> getSPIExtensionClasses() {
-        return Collections.<String>emptySet();
+    public Set<Supplier<Object>> getSPIExtensionSuppliers() {
```

Comment:
I know it is borderline but I think this is a functional change that should have a copyright update